src/database.py

```
@@ -2,15 +2,15 @@
 from loguru import logger
 from config import settings
 from typing import List, Dict, Any, Optional
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)